inst/www/css/disconnect-dialogue.css

```diff
@@ -0,0 +1,36 @@
+#ss-connect-dialog { display: none !important; }
+#shiny-disconnected-overlay { display: none !important; }
+
+#ss-overlay {
+             background-color: #000000 !important;
+             opacity: 0.6 !important;
+             position: fixed !important;
+             top: 0 !important;
+             left: 0 !important;
+             bottom: 0 !important;
+             right: 0 !important;
+             z-index: 99998 !important;
+             overflow: hidden !important;
+             cursor: not-allowed !important;
+          }
+
+#custom-disconnect-dialog {
+             background: #ffffff !important;
+             color: #000000 !important;
+             width: full !important;
+             transform: translateX(-50%) translateY(-50%) !important;
+             font-size: 16px !important;
+             top: 50% !important;
+             position: fixed !important;
+             bottom: auto !important;
+             left: 50% !important;
+             padding: 0.8em 1.5em !important;
+             text-align: center !important;
+             height: auto !important;
+             opacity: 1 !important;
+             z-index: 99999 !important;
+             border-style: solid !important;
+             border-color: #ffdd00 !important;
+             border-width: 0.25em !important;
+             box-shadow: rgba(0, 0, 0, 0.3) 3px 3px 10px !important;
+          }
```

Comment:
Could we test this without the `!important` after every line? The top ones might need it to force the standard shiny stuff to be hidden but's generally bad practice to better to avoid doing if it's not needed

Reply:
Tested out removing all the importants from the css and running with the test dashboard and all seems to still work fine.